### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by Pradhvan**
* Added three templates for bug_report, feature_request and how to
question.

Closes https://github.com/scanapi/scanapi/issues/382

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/how_to_question.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### What you intend to do with scanapi ?
+<!-- A clear and concise description of what you want to with scanapi. -->
+
+### Expected behavior ?
+<!-- A clear and concise description of what you expected to happen. -->
+
+
+---
+
+scanapi has a [discord server](#) so feel free to join and say 👋 hello. Maybe share the issue link link?
```
Comment (camilamaia) [inline .github/ISSUE_TEMPLATE/how_to_question.md:22]: @Pradhvan I've created an invite link that never expires, so we can use it: `https://discord.gg/f3cCXJ2ZPB`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: 🐛 Create a a bug report.
+labels: bug, needs triage
```
Comment (camilamaia) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: 👏 I've create the label [`needs triage`](https://github.com/scanapi/scanapi/labels/needs%20triage) for us!

**Review by camilamaia — APPROVED** [review]

**Comment by camilamaia** [comment]
Hey @Pradhvan, since you have been helping us a LOT with ScanAPI, we've promoted you to a `Maintainer` 🎉  Now you have the `maintain` permission for our repos! We still don't have official docs explaining more about this role, but we definitely should soon (And probably also some other docs to explain how to apply for roles). We just wanted to promote you and we thought we should not postpone it

But please keep in mind, as we said in our [CONTRIBUTING guide](https://github.com/scanapi/contributors/blob/main/CONTRIBUTING.md):

> Do not ever feel bad for not contributing to open source.

This is our way to say thanks for all the effort you have ALREADY done for ScanAPI 😻.

Thank you again!! 🎉

File: .github/ISSUE_TEMPLATE/how_to_question.md (hunk 1 above)
Comment (Pradhvan) [reply .github/ISSUE_TEMPLATE/how_to_question.md:22]: Great! I Will update that. 😄

**Comment by Pradhvan** [comment]
> We just wanted to promote you and we thought we should not postpone it

@camilamaia thank you so much. ❤️ 

Excited to be part of the team. 🙌🏾

**Comment by Pradhvan** [comment]
@camilamaia updated the discord server link. Do I need to add a changelog too?

Besides that, I think it's good to go 🚀  Let's merge this. :octocat:

File: .github/ISSUE_TEMPLATE/how_to_question.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+### What you intend to do with scanapi ?
+<!-- A clear and concise description of what you want to with scanapi. -->
+
+### Expected behavior ?
+<!-- A clear and concise description of what you expected to happen. -->
+
+
+---
+
+scanapi has a [discord server](https://discord.gg/f3cCXJ2ZPB) so feel free to join and say 👋 hello. Maybe share the issue link link?
```
Comment (Pradhvan) [inline .github/ISSUE_TEMPLATE/how_to_question.md:22]: I Will fix this typo 😄

**Comment by camilamaia** [comment]
@Pradhvan Perfect, let me merge it! Thanks for the hard work 💪 

No need for a changelog entry, since this is only internal changes, it is not a change that effect users of ScanAPI.
https://github.com/scanapi/scanapi/wiki/Changelog#what-warrants-a-changelog-entry

**Pull request merged**